src/oc/hoc_oop.cpp

```diff
@@ -1304,10 +1309,10 @@ void hoc_object_component() {
                     if (narg == 1) {
                         hoc_push_ndim(1);
                     } else {
-                        hoc_execerr_ext("%s.%s is array not function. Use %s[...] syntax",
```

Comment:
Good, `_ext` is the C-style equivalent of `_fmt` AFAICT.

Reply:
yes.